src/stories/markdowns/lists.mdx

```diff
@@ -1,13 +1,15 @@
-1. 1\. Plain
-2. 2\. With `inline code`
+2. 2\. Plain
+3. 3\. With `inline code`
 
 ---
 
 1. First item
 2. Second item
 3. Third item
-   4. Indented item 1
-   5. Indented item 2
```

Comment:
So it seems

1. This snapshot was depending on us not supporting `start`
1. A blank line is required to separate the paragraph and nested list in the latest Remark version